src/mudlet.pro

```diff
@@ -104,22 +111,25 @@ VERSION = 4.4.0
 # if you are distributing modified code, it would be useful if you
 # put something distinguishing into the MUDLET_VERSION_BUILD environment
 # variable to make identification of the used version simple
-# the qmake BUILD variable is NOT built-in
-BUILD = $$(MUDLET_VERSION_BUILD)
-isEmpty( BUILD ) {
-# Possible values are:
+ENV_MUDLET_VERSION_BUILD = $$(MUDLET_VERSION_BUILD)
+isEmpty( ENV_MUDLET_VERSION_BUILD ) {
+# Possible {reserved by the Mudlet Makers team} values are:
 # "-dev" for the development build
 # "-public-test-build" for the public test build
 # "" for the release build
-   BUILD = "-dev"
+# This forcibly prevents a build being identified internally as a release
+# version as it requires manual editing - usually by the Project leader - to
+# temporarily clear this value when a release is being produced:
+   ENV_MUDLET_VERSION_BUILD = "-dev"
 }
 
-# Changing BUILD and VERSION values affects: ctelnet.cpp, main.cpp, mudlet.cpp
-# dlgAboutDialog.cpp and TLuaInterpreter.cpp.  It does NOT cause those files to
-# be automatically rebuilt so you will need to 'touch' them...!
+# Changing MUDLET_VERSION_BUILD and VERSION values affects: ctelnet.cpp,
+# main.cpp, mudlet.cpp, dlgAboutDialog.cpp and TLuaInterpreter.cpp.  It does NOT
+# cause those files to be automatically rebuilt so you will need to 'touch'
```

Comment:
`'touch'` means nothing to a Windows developer, so for simplicity's sake we can just say you should rebuild the project.

Reply:
https://en.wikipedia.org/wiki/Touch_(command)

Well rebuilding the *whole* project is a bit of overkill and can take a while for those not on a powerful machine (when I started hacking Mudlet six years ago my PC took 45 minutes to do a full build!) - so identifying which files *do* need rebuilding after a change can save *some* people *some* time... :grinning: